src/main/main.cpp

```diff
@@ -12,11 +12,24 @@
 
 INITIALIZE_EASYLOGGINGPP
 
+namespace stellar
+{
+static void
+outOfMemory()
+{
+    std::cerr << "Unable to allocate memory \n";
```

Comment:
you have to use `std::endl` in order to flush

also, I think you need to include `<cstdlib>` for the call to `std::abort` to be portable

Reply:
done